core/src/test/java/org/apache/calcite/test/MaterializedViewSubstitutionVisitorTest.java

```diff
@@ -1607,6 +1607,66 @@ private void checkSatisfiable(RexNode e, String s) {
             + "EnumerableTableScan(table=[[hr, MV0]])")).ok();
   }
 
+  @Test void testGroupSetLiteral1() {
```

Comment:
Maybe the test name can hint towards the specific SQL query that is tested. 

For instance, here you simply have a constant followed by a column name, the test name could be `testGroupSetLiteralBeforeColumnName`, test `testGroupSetLiteral2` could be `testGroupSetMultipleLiteralsBeforeColumnName`, etc.

In this way, whenever a test failure occurs, already the name of the test provides a hint towards which functionality was broken.

WDYT?

Reply:
Thanks for review.The `order` is not very important.and I updated the name of the tests.

Reply:
LGTM, I have just left a small additional comment :)